ludwig/api.py

```diff
@@ -72,8 +72,7 @@
 
 class LudwigModel:
     def __init__(self,
-                 model_definition=None,
-                 model_definition_fp=None,
+                 model_definition,
```

Comment:
Nice!  I had thought about making this change, so glad you went ahead and did it!